docs/01-utilities.md

```diff
@@ -0,0 +1,49 @@
+# Utilities
+The following utilities need to be installed in order to follow Kepler installation guide:
+
+- Go
+- Make
+- Docker
+- kubectl
+- Helm
+
+# Installation Steps
+
+## Installing Go
+
+```bash
+sudo apt install golang-go
+```
+
+## Installing Make
+
+```bash
+sudo apt install make
+```
+
+## Installing Docker 
+
+```bash
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg lsb-release 
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg

```

Comment:
I had to add these steps - tried this in a Vagrant+Virtualbox env in my attempt to get Kepler to work on MacOs 😅 
```suggestion
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
sudo chmod a+r /etc/apt/keyrings/docker.gpg
echo \
  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
```